src/Microsoft.VisualStudio.AppDesigner/Common/ShellUtil.vb

```diff
@@ -186,7 +186,7 @@ Namespace Microsoft.VisualStudio.Editors.AppDesCommon
             Const ProjectsAndSolution As String = "ProjectsandSolution"
 
             Try
-                ProjAndSolutionProperties = DTE.Properties(EnvironmentCategory, ProjectsAndSolution)
+                ProjAndSolutionProperties = DTE.get_Properties(EnvironmentCategory, ProjectsAndSolution)
```

Comment:
Just for my understanding, why did a bunch of these need to use explicit `get_` calls?

Reply:
I kept seeing these errors:
```
G:\project-system\src\Microsoft.VisualStudio.AppDesigner\Common\ShellUtil.vb(189,45): error BC30456: 'Properties' is not a member of 'DTE'. [G:\project-system\src\Microsoft.VisualStudio.AppDesigner\Microsoft.VisualStudio.AppDesigner.vbproj]
G:\project-system\src\Microsoft.VisualStudio.AppDesigner\PropPages\PropPageUserControlBase.vb(2836,20): error BC30456: 'FileNames' is not a member of 'ProjectItem'. [G:\project-system\src\Microsoft.VisualStudio.AppDesigner\Microsoft.VisualStudio.AppDesigner.vbproj]
```
Ryan Molden then replied with this: Properties and FileNames are both named indexers, they were converted to get_Properties and get_FileNames in the PIA you are consuming. There is ongoing work to provide a way to NOT have to convert named indexers to get_ equivalents, but it is what must be done today.